README.md

```diff
@@ -55,9 +55,9 @@ Now the data will be populated within your database, but it won't be connected i
 mm = ModelManager()
 mm.create_model_from_data()
 ```
-Here we can see we now have class and relationship nodes, illustrating connections between Study, Subject, Age and Sex. 
+Here we can see we now have class and relationship nodes, illustrating connections between Study, Subject, Age and Sex. The red nodes below indicate relationship nodes, while the blue indicate class nodes.
 
-![modelmanager example](examples/data/create_model_from_data%20example.PNG)
+![modelmanager example](examples/data/modelmanager_example.PNG)
```

Comment:
@shannonhaughton I think you need to "unpack all red nodes", so that we see all the Classes. Also may make the Class nodes a little bigger